web/cobrands/northamptonshire/assets.js

```diff
@@ -483,4 +483,30 @@ fixmystreet.assets.add($.extend(true, {}, northants_road_defaults, {
     ]
 }));
 
+// Hide form when emergency category used
+function check_emergency() {
+    var relevant_body = OpenLayers.Util.indexOf(fixmystreet.bodies, northants_defaults.body) > -1;
+    var relevant_cat = $('label[for=form_emergency]').length;
+    var relevant = relevant_body && relevant_cat;
+    var currently_shown = !!$('#northants-emergency-message').length;
+    var body = $('#form_category').data('body');
+
+    if (relevant === currently_shown || body) {
+        // Either should be shown and already is, or shouldn't be shown and isn't
+        return;
+    }
+
+    if (!relevant) {
+        $('#northants-emergency-message').remove();
+        $('.js-hide-if-invalid-category').show();
+        return;
+    }
+
+    var $msg = $('<div class="box-warning" id="northants-emergency-message"></div>');
+    $msg.html($('label[for=form_emergency]').html());
```

Comment:
I still seem to get both this added message and the standard extra information displayed.

![Screenshot 2019-03-29 at 12 27 31](https://user-images.githubusercontent.com/5310/55232641-15380480-521e-11e9-8541-033a8ab5c31c.png)

Reply:
Oh right, yes, this will only be hidden once #2433 is merged (which I should do now) because that then puts the extra questions inside `js-hide-if-invalid-category`.

Reply:
That's now merged, I've rebased this on top so this should be okay now.

Reply:
One other thing, if I select repeated emergency categories I see the message multiple times, so e.g. if I select Drains -> Blocked flooding private property and then Blocked flooding road/path I get the message twice.

Reply:
Grr, stupid booleans. Should be okay now!